file_paths.md

```diff
@@ -252,6 +277,10 @@ user or group.
         lib *
             puppet.rb *
 
+    C:\Program Files\Puppet Labs\Puppet\pxp-agent *
+        modules *                             # contains pxp-agent modules (ruby binaries)
+            pxp-module-puppet *               # executes Puppet and provides results of Puppet runs
```

Comment:
Yes that's the file. Once this PR is agreed on and merged, we'll start updating code where there are mismatches.